### PR TITLE
Update french translation

### PR DESCRIPTION
**Pull request by jeau**
Hi @kevinpapst 

I have done some corrections and also added properties that were missing from the first set of messages I worked on yesterday.

### CODE REVIEW

**Comment by kevinpapst** [comment]
Thanks!

I will wait a bit before merging, so you can test it for one or two days to see if you find more stuff ;-)

File: translations/messages.fr.xliff [hunk 1]
```diff
@@ -129,7 +137,7 @@
             -->
             <trans-unit id="label.date">
                 <source>label.date</source>
-                <target>Date</target>
+                <target>Date </target>
```
Comment (kevinpapst) [inline translations/messages.fr.xliff]: Why / where is that space needed? Probably its better to adjust the template if there needs to be an additional space. In that case its likely that every language is affected. 
Or is that space by accident?

Comment (jeau) [reply translations/messages.fr.xliff]: in French we put a space before the colon, in fact before all "double punctuation" :  ; !  ? 

I can put a &nbsp; entitie on this place

**Comment by jeau** [comment]
with non-breaking space entitie, during install the composer command render this error:

```
Executing script cache:clear [KO]
 [KO]
Script cache:clear returned with error code 1
!!
!!   // Clearing the cache for the prod environment with debug
!!   // false
!!
!!
!!  In XliffFileLoader.php line 56:
!!
!!    Unable to load "/var/www/html/translations/messages.fr.xliff": [ERROR 522]
!!    Validation failed: no DTD found ! (in n/a - line 2, column 67)
!!    [ERROR 26] Entity 'nbsp' not defined (in n/a - line 140, column 35)
!!    [ERROR 26] Entity 'nbsp' not defined (in n/a - line 468, column 37)
!!    [ERROR 26] Entity 'nbsp' not defined (in n/a - line 711, column 39)
!!    [ERROR 26] Entity 'nbsp' not defined (in n/a - line 723, column 44)
!!
!!
!!  In XmlUtils.php line 62:
!!
!!    [ERROR 522] Validation failed: no DTD found ! (in n/a - line 2, column 67)
!!    [ERROR 26] Entity 'nbsp' not defined (in n/a - line 140, column 35)
!!    [ERROR 26] Entity 'nbsp' not defined (in n/a - line 468, column 37)
!!    [ERROR 26] Entity 'nbsp' not defined (in n/a - line 711, column 39)
!!    [ERROR 26] Entity 'nbsp' not defined (in n/a - line 723, column 44)
!!
!!
!!  cache:clear [--no-warmup] [--no-optional-warmers] [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-e|--env ENV] [--no-debug] [--] <command>
!!
!!
```

**Comment by SteamWind** [comment]
Why did you put nbsp?

**Comment by jeau** [comment]
to leave an unbreakable space before the colon, but that's not a good idea.

cf. https://github.com/kevinpapst/kimai2/pull/279/files/57adf409f8f53f13d1c72353cea953aa3b96927b#diff-568861cebf7cd8c1918b903990b22b9e

you're right, it's probably best to add special translation keys

**Comment by kevinpapst** [comment]
@jeau Done?

**Comment by kevinpapst** [comment]
Perfect, thanks!

I am quite busy currently ... the other issues will be solved as well, but I need a week or two for other stuff before that.

File: translations/messages.fr.xliff (hunk 1 above)
Comment (SteamWind) [reply translations/messages.fr.xliff]: I see the point. I think it's not as important...
I don't usually put this space before those punctuations in french but it looks better.

**Pull request merged**